docs/source/how-to/tiled-database.md

```diff
@@ -34,7 +34,8 @@ in place of `...` below. Inject the password via an environment variable as show
 do not hard-code it in the configuration file. Be sure to quote the URI.
 
 ```yaml
-database_uri: "postgresql://tiled:${TILED_DATABASE_PASSWORD}@.../tiled"
+database
+  uri: "postgresql://tiled:${TILED_DATABASE_PASSWORD}@.../tiled"
```

Comment:
Should this describe pool size and pre pool ping?

Reply:
Good note.

I added a section to the bottom of this page that links to the service configuration reference, which already includes descriptions of the various options.